### PR TITLE
Testing instructions

### PR DESCRIPTION
**Pull request by GeorgianaElena**
The instructions should work after https://github.com/2i2c-org/pilot-hubs/pull/130 is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +39,27 @@ You can run a local JupyterHub to test your template changes.
...
+   ```
+3. It should take around 5min to see your changes on the staging hub at `https://staging.pilot.2i2c.cloud`.
+
+4. After you've checked that everything works, merge the `staging` branch into master and the changes
+will be deployed to the other hubs too after around 5min.
+
+5. Delete the remote staging branch, either from the GitHub GUI, or using:
+
+   ```bash
+   git branch -D staging
```
Comment (yuvipanda) [inline README.md]: This only deletes local branch, right? I thought deleting remote branch required something like `git push -d <remote> <branchname>`

File: templates/error.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		padding: 30px;
+		margin: 0 auto;
+		overflow: hidden;
+		font-size: 17px;
+	}
+</style>
+<div class="error">
+	<div class="info">
+		<i class="fa fa-info-circle"></i>
+	  	If your email address has <b>NOT</b> been added to the list of allowed users for this hub, please <b>contact</b> the hub administrators.
```
Comment (yuvipanda) [inline templates/error.html]:
```suggestion
	  	Looks like you have <b>NOT</b> been added to the list of allowed users for this hub. Please contact the hub administrators.
```

Comment (yuvipanda) [reply templates/error.html]: GitHub auth doesn't use email, only usernames. This makes the wording more generic

File: README.md (hunk 1 above)
Comment (GeorgianaElena) [reply README.md]: Oh, right! Checking the git branch man page, looks like `-D` is only to force the local branch deletion :facepalm: Thank yoou! <3

**Pull request merged**